src/System.Composition.Hosting/src/System/Composition/Hosting/Providers/ExportFactory/ExportFactoryExportDescriptorProvider.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace System.Composition.Hosting.Providers.ExportFactory
 {
-    internal class ExportFactoryExportDescriptorProvider : ExportDescriptorProvider
+    public class ExportFactoryExportDescriptorProvider : ExportDescriptorProvider
```

Comment:
Yes , we do have a way to bypass the test against bug for UAPAOT. 

> This change is introducing a new public APIs, and so it would need to go through new API review process.
I don't think we have enough justification to get it  through API review process , "we want to get to zero test failure for helix uwp run" probably wont cut it. Also i don't know of any UWP app which use System.Composition.Hosting.  


I think the best course of action here is 
1. Open the VSO bug for linker/DR control feature
2. Disable the test for uapaot for now and enable the test when we have the linker/DR feature.